ecs-agent/acs/session/task_manifest_responder.go

```diff
@@ -83,6 +83,12 @@ func (tmr *taskManifestResponder) handleTaskManifestMessage(message *ecsacs.Task
 	logger.Debug(fmt.Sprintf("Processing %s", TaskManifestMessageName), logger.Fields{field.MessageID: message.MessageId})
 	m := tmr.metricsFactory.New(metrics.TaskManifestHandlingDuration)
 	err := tmr.processTaskManifestMessage(message)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Unable to handle %s", TaskManifestMessageName), logger.Fields{
+			field.MessageID: aws.StringValue(message.MessageId),
```

Comment:
Because we're now using message.MessageId in two places here(L83), maybe declare a var to hold `aws.StringValue(message.MessageID)`?

Reply:
Sure, can do!